massdriver.yaml

```diff
@@ -2,8 +2,8 @@ schema: draft-07
 name: "azure-fhir-service"
 description: "Azure FHIR Service enables rapid exchange of health data using the Fast Healthcare Interoperability Resources (FHIR) data standard."
 source_url: github.com/massdriver-cloud/azure-fhir-service
-access: "private"
-type: "bundle"
+access: public
```

Comment:
why public?

Reply:
I thought this is what we set up for our open sourced massdriver bundles. I can change back to `private`, no problem